gwtp-core/gwtp-dispatch-rest/src/main/java/com/gwtplatform/dispatch/rest/client/core/DefaultBodyFactory.java

```diff
@@ -127,11 +124,16 @@ private SerializedValue getSerializedValue(RestAction<?> action, Object object)
             Serialization serialization = findSerialization(bodyClass, contentTypes);
 
             if (serialization != null) {
-                return serialize(serialization, object, bodyClass, contentTypes);
+                try {
```

Comment:
I agree it was moved out when I was using the action as a parameter argument, but I removed it so I could move it back.